.github/workflows/actions.yml

```diff
@@ -0,0 +1,62 @@
+name: Actions Workflow
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  build-project: #First Job is to check the code formatting, linting and build the project.
+    name: Run Prettier, Lint and Build Checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Code Formatting Check #Runs Prettier Script
+        run: npm run prettier
+
+      - name: Linting Check #Runs ESLint Script
+        run: npm run lint
+
+      - name: Build #Build the project
+        run: npm run build
+
+      - name: Check Build Status
+        run: |
+          if [ $? -eq 0 ]; then
+            echo "Build successful!🎉"
+          else
+            echo "Build failed!"
+            exit 1
+          fi
+
+  chromatic-deployment: #Second Job is to deploy the project to Chromatic
+    # Operating System
```

Comment:
Please add the `name` attribute over here.

Reply:
done!